app/services/foreman/access_permissions.rb

```diff
@@ -393,6 +393,9 @@
     map.permission :edit_classes,   {:host_editing => [:edit_classes],
                                      :"api/v2/host_classes" => [:index, :create, :destroy]
                                     }
+    map.permission :view_params,  {:host_editing => [:view_params],
```

Comment:
I don't see how this particular route is used.  Is the patch missing corresponding code in the view?